scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -4,6 +4,17 @@ scaffolding_load() {
   _setup_funcs
   _setup_vars
 
+# /src is the default value for $SRC_PATH
+# if it is set to something else
+# this scaffolding will not work
+if [ "$SRC_PATH" = "/src" ]
```

Comment:
You could also use `if [ "$SRC_PATH" != "/src" ]; then` which gets you the negative case

Reply:
Agreed and updated!